stream-chat-android/src/main/java/com/getstream/sdk/chat/view/MessageListView.kt

```diff
@@ -160,6 +162,19 @@ public class MessageListView : ConstraintLayout {
                 onMessageFlagHandler
             ).show()
         }
+    //Todo: Add this after the behaviour of message options is complete.
```

Comment:
If I put this... then I would change the original behaviour before the new one is completed... Not sure if the comment is the best solution, thought...

Reply:
Removrd